hudi-common/src/main/java/org/apache/hudi/common/config/HoodieCommonConfig.java

```diff
@@ -41,6 +41,13 @@ public class HoodieCommonConfig extends HoodieConfig {
       .noDefaultValue()
       .withDocumentation("The query instant for time travel. Without specified this option, we query the latest snapshot.");
 
+  public static final ConfigProperty<Integer> TABLE_LOADING_PARALLELISM = ConfigProperty
+      .key("hoodie.datasource.read.loading.parallelism")
+      .defaultValue(-1)
```

Comment:
we don't normally use -1 to disable a feature. usually we have a xxxEnabled. and we should mark this advanced() config. and can we auto infer it somehow?